management/src/test/java/org/ehcache/management/providers/statistics/StatsUtil.java

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
it was in the management test folder but I needed to access it from both management tests and :clustered:integration-test.  The ClusteringManagementServiceTest wasn't able to see it from the management test folder.  Is there a way to change the gradle.build to see it?